### ISSUE TITLE
a11y: column restricted mode for full-screen applications

### ISSUE THREAD

**Issue by kruton** [connectbot/termlib#31]
For applications like Midnight Commander that split the window into multiple panes, reading line-by-line does not make sense. A custom accessibility action should be added to create a column mode that tries to detect visually distictive splits using the pipe character or some other heuristic.
